idunn/blocks/wikipedia.py

```diff
@@ -5,7 +5,67 @@
 from requests.exceptions import HTTPError, RequestException, Timeout
 import pybreaker
 from elasticsearch import Elasticsearch
+from redis import ConnectionPool, ConnectionError, TimeoutError
 
+from redis_rate_limit import RateLimiter, TooManyRequests
+
+class HTTPError40X(HTTPError):
+    pass
+
+class WikipediaLimiter:
+    _limiter = None
+    _redis_url = None
+
+    @classmethod
+    def get_limiter(cls):
+        if cls._limiter is None:
+            from app import settings
+
+            cls._redis_url = settings['WIKI_API_REDIS_URL']
+
+            if cls._redis_url is not None:
+                """
+                If a redis url has been provided to Idunn,
+                then we use the corresponding redis
+                service in the rate limiter.
+                """
+                url = cls._redis_url.split(":")[0]
+                port = cls._redis_url.split(":")[1]
+
+                max_calls= settings['WIKI_API_RL_MAX_CALLS']
```

Comment:
it does not seems to work in real like as when read from the config this is a string and this results in error like
```
  File "/dev/idunn/idunn/blocks/wikipedia.py", line 61, in wrapper
    with limiter.limit(client="Idunn"):
  File "/.local/share/virtualenvs/idunn-OLqrlkmx/lib/python3.6/site-packages/redis_rate_limit/__init__.py", line 69, in __enter__
    self.increment_usage()
  File "/.local/share/virtualenvs/idunn-OLqrlkmx/lib/python3.6/site-packages/redis_rate_limit/__init__.py", line 122, in increment_usage
    if int(current_usage) > self._max_requests:
TypeError: '>' not supported between instances of 'int' and 'str'
```

Reply:
done